apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -183,7 +185,13 @@ private void publishInsertEvent(
     JSONObject partitionKeyValuesObject = new JSONObject(partitionKeyValues);
     json.put("partitionKeyValues", partitionKeyValuesObject);
 
-    sendMessage(json);
+    sendMessage(json, getMessageAttributes(eventType));
+  }
+
+  private Map<String, MessageAttributeValue> getMessageAttributes(EventType eventType) {
+    Map<String, MessageAttributeValue> map = new HashMap<>();
+    map.put("eventType", new MessageAttributeValue().withStringValue(eventType.toString()));
```

Comment:
Enum please: maybe `MessageAttributeKeys`.
Can we have a little section in the README where we list the keys that can be filtered on?